doc/SpectatorView.Setup.DSLR.md

```diff
@@ -1 +1,164 @@
+# Requirements
+
+> Note: DSLR Filming currently requires a HoloLens 2 for the camera rig. HoloLens 1 devices are not supported.
+
+
+1. HoloLens 2 (This device will be needed in addition to the device worn by the user)
+2. Windows PC
+3. Visual Studio 2017 installed on the PC
+4. Windows 10 SDK (10.0.18362.0)
+5. Unity installed on the PC
+6. Digital single-lens reflex (DSLR) camera

```

Comment:
>Digital single-lens reflex (DSLR) camera [](start = 3, length = 40)

I think technically any video camera that can output 1080p video over SDI or HDMI will work, the sensor type is not important.